File: HyeonHoLe2/1주차/PG_131705_삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int result = 0;
        for(int i=0; i<number.length; i++){
            for(int j=i+1; j<number.length; j++){
                for(int k=j+1; k<number.length; k++){
                    answer = number[i] + number[j] + number[k];
                    if(answer==0){
                        result++;
                    }
                }
                
            }
            
        }
        return result;
    }
}

File: Seoin0319/1_week/PG_131705_삼총사.py
def solution(number):
    answer = 0
    n = len(number)

    # 서로 다른 학생 3명을 뽑기 위해
    # i < j < k 순서로 모든 조합을 확인
    for i in range(n):
        for j in range(i + 1, n):
            for k in range(j + 1, n):
                # 세 학생의 번호 합이 0이면 삼총사
                if number[i] + number[j] + number[k] == 0:
                    answer += 1

    return answer